### PR TITLE
Added package perl-math-basecnv

### PR DESCRIPTION
**Pull request by sergey-safarov**

### CODE REVIEW

File: main/perl-math-basecnv/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+url="http://search.cpan.org/dist/Math-BaseCnv/"
+arch="noarch"
+license="GPL-3.0"
+depends="perl"
+makedepends="perl-dev"
+install=""
+subpackages="$pkgname-doc"
+source="http://search.cpan.org/CPAN/authors/id/P/PI/PIP/Math-BaseCnv-$pkgver.tgz"
+
+_builddir="$srcdir"/Math-BaseCnv-$pkgver
```
Comment (Ikke) [inline main/perl-math-basecnv/APKBUILD]: `builddir` is now officially supported, so it's not required to start with an `_` anymore.

File: main/perl-math-basecnv/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+pkgname=perl-math-basecnv
+pkgver=1.14
+pkgrel=0
+pkgdesc="Math::BaseCnv perl module"
+url="http://search.cpan.org/dist/Math-BaseCnv/"
+arch="noarch"
+license="GPL-3.0"
+depends="perl"
+makedepends="perl-dev"
+install=""
```
Comment (Ikke) [inline main/perl-math-basecnv/APKBUILD]: Could you please remove empty variables?

File: main/perl-math-basecnv/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+subpackages="$pkgname-doc"
+source="http://search.cpan.org/CPAN/authors/id/P/PI/PIP/Math-BaseCnv-$pkgver.tgz"
+
+_builddir="$srcdir"/Math-BaseCnv-$pkgver
+prepare() {
```
Comment (Ikke) [inline main/perl-math-basecnv/APKBUILD]: `default_prepare` handles applying patches now. Options can be supplied through `$patch_args`.

File: main/perl-math-basecnv/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+	for i in $source; do
+		case $i in
+		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
+		esac
+	done
+}
+
+build() {
+	cd "$_builddir"
+	PERL_MM_USE_DEFAULT=1 perl Makefile.PL INSTALLDIRS=vendor || return 1
```
Comment (Ikke) [inline main/perl-math-basecnv/APKBUILD]: `|| return 1` is no longer required.

**Comment by sergey-safarov** [comment]
Hello @Ikke 
Please review last PC.
If all ok, then i will squash commits

**Comment by sergey-safarov** [comment]
Hi @Ikke 
Could you make review of this PR?

File: main/perl-math-basecnv/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# Contributor: Natanael Copa <ncopa@alpinelinux.org>
+# Maintainer: Natanael Copa <ncopa@alpinelinux.org>
```
Comment (Ikke) [inline main/perl-math-basecnv/APKBUILD]: Is ncopa aware that you put him as maintainer? If you contribute this package, you most likely will have to maintain this package yourself.

File: main/perl-math-basecnv/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+subpackages="$pkgname-doc"
+source="http://search.cpan.org/CPAN/authors/id/P/PI/PIP/Math-BaseCnv-$pkgver.tgz"
+options="!check"
+
+builddir="$srcdir"/Math-BaseCnv-$pkgver
+
+build() {
+	cd "$builddir"
+	PERL_MM_USE_DEFAULT=1 perl Makefile.PL INSTALLDIRS=vendor
+	make && make test || return 1
```
Comment (Ikke) [inline main/perl-math-basecnv/APKBUILD]: `make test` can be moved to a `check()` function. Then you can remove `options="!check"`

`|| return 1` is no longer necessary

**Comment by sergey-safarov** [comment]
@Ikke PR is updated.

**Pull request closed without merging**